hops/util.py

```diff
@@ -167,8 +169,25 @@ def _parse_rest_error(response_dict):
         user_msg = response_dict[constants.REST_CONFIG.JSON_USR_MSG]
     return error_code, error_msg, user_msg
 
+def get_secret(project_name, secrets_store, secret_key):
```

Comment:
Is it a public method? If so add docstring. If not, prepend "_" to the methodname.

Reply:
It's public. I added a doc string.